packages/synthetix-main/contracts/modules/core/PoolModule.sol

```diff
@@ -29,10 +28,6 @@ contract PoolModule is IPoolModule {
             revert AddressError.ZeroAddress();
         }
 
-        if (Pool.exists(requestedPoolId)) {
-            revert PoolAlreadyExists(requestedPoolId);
```

Comment:
Why is this removed here?

Reply:
The check was moved from PoolModule to Pool, to here:
https://github.com/Synthetixio/synthetix-v3/pull/1153/files#diff-f5317ceb232ae692021922bfaf85dbbce8bf1208cab93465eaa7a5634e2f8334R118-R121